### DISCUSSION TITLE
Apply Issue

### DISCUSSION THREAD

**Discussion by elilin0610** [community/community#145092] [GitHub Education]
### Hi everyone,

Hi There,

My question is should I change the Public Profile's name to my legal name or I just need to complete my GitHub billing information with my legal name, sorry if I make something wrong in English am not native.

![未命名](https://github.com/user-attachments/assets/3b0a38a8-256e-4703-98c8-f9698f2d2e18)

Following is email from GitHub Education

Please complete your [GitHub billing information](https://github.com/settings/billing/payment_information) with your full name exactly as it appears in your academic affiliation document. You do not have to add a payment method. You may need to log out and log back in to GitHub before reapplying. If you have only a single legal name, enter it in both the first and last name fields.

The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.

,Thanks

**Comment by JoaoCoelho2003** [comment]
Hi!
It's important to align your GitHub user profile and billing information with the details on your academic affiliation document. Based on the instructions from GitHub Education:

- **GitHub Billing Information:** Your billing information must include your full legal name, as it appears in your academic documents. However, you do not need to add a payment method.

- **GitHub Public Profile Name:** While it isn’t mandatory for your GitHub public profile name to match your legal name, most people choose to include their legal name in some form.
